client/src/components/Intro/Intro.test.js

```diff
@@ -12,20 +12,24 @@ function rendererCreateWithRedux(ui) {
 }
 
 describe('<Intro />', () => {
-  it('has no blockquotes when loggedOut', () => {
+  it('has no blockquotes when loggedOut', async () => {
```

Comment:
Hi @anirudh-modi, this is a bit of weird one.  Unfortunately, the new lint rules are throwing false positives here.  They're interpreting `react-test-renderer` functions as `@testing-library/react` functions.

What I recommend is to revert these changes and instead leave a comment explaining why the lint rules had to be disabled for this case.

Reply:
Thanks for catching this, Oliver. I missed this one.

Reply:
No worries.  You've done an amazing job QAing this, so thanks again.

Reply:
Hi @ojeytonwilliams thanks for your feedbacks, let me have a look at the same.. on the false postiive.. @huyenltnguyen Let me update my branch with `main` as well

Reply:
@huyenltnguyen I have also updated my branch with `main` and @ojeytonwilliams I have disabled the eslint for the given test with appropriate comment.